usaspending_api/etl/management/commands/es_configure.py

```diff
@@ -36,69 +44,87 @@ def add_arguments(self, parser):
+
+    def get_index_template(self):
+        template = self.return_json_from_file(FILES[self.template])
+        template["index_patterns"] = [self.index_pattern]
+        template["settings"]["index.max_result_window"] = self.max_result_window
+        self.validate_known_fields(template)
+        return template
+
+    def return_json_from_file(self, path):
+        """Read and parse file as JSON
+
+        Library performs JSON validation which is helpful before sending to ES
+        """
+        filepath = str(path)
+        if not path.exists():
+            raise SystemExit("Fatal error: file {} does not exist.".format(filepath))
+
+        print("Reading file: {}".format(filepath))
+        with open(filepath, "r") as f:
+            json_to_dict = json.load(f)
+
+        return json_to_dict
+
+    def validate_known_fields(self, template):
+        defined_fields = set(
+            [field for field in template["mappings"]["{view}_mapping".format(view=self.type[:-1])]["properties"]]
+        )
+        load_columns = set(self.load_columns)
+        if defined_fields ^ load_columns:  # check if any fields are not in both sets
+            raise RuntimeError("Mismatch between template and fields in ETL! Resolve before continuing!")
 
-def get_elasticsearch_settings():
-    es_config = return_json_from_file(FILES["settings"])
-    es_config["settings"]["index.max_result_window"] = settings.ES_TRANSACTIONS_MAX_RESULT_WINDOW
-    return es_config["cluster"], es_config["settings"]
 
+def retrieve_transaction_index_template():
+    """This function is used for test configuration"""
+    with open(str(FILES["transaction_template"])) as f:
+        mapping_dict = json.load(f)
+        template = json.dumps(mapping_dict)
 
-def get_index_template():
-    template = return_json_from_file(FILES["template"])
-    template["index_patterns"] = ["*{}".format(settings.ES_TRANSACTIONS_NAME_SUFFIX)]
-    template["settings"]["index.max_result_window"] = settings.ES_TRANSACTIONS_MAX_RESULT_WINDOW
-    validate_known_fields(template)
     return template
 
 
-def return_json_from_file(path):
-    """Read and parse file as JSON
-
-    Library performs JSON validation which is helpful before sending to ES
-    """
-    filepath = str(path)
-    if not path.exists():
-        raise SystemExit("Fatal error: file {} does not exist.".format(filepath))
-
-    print("Reading file: {}".format(filepath))
-    with open(filepath, "r") as f:
-        json_to_dict = json.load(f)
-
-    return json_to_dict
-
-
-def validate_known_fields(template):
-    defined_fields = set([field for field in template["mappings"]["transaction_mapping"]["properties"]])
-    load_columns = set(VIEW_COLUMNS)
-    if defined_fields ^ load_columns:  # check if any fields are not in both sets
-        raise RuntimeError("Mismatch between template and fields in ETL! Resolve before continuing!")
-
-
-def retrieve_transaction_index_template():
+def retrieve_award_index_template():
```

Comment:
Why is this a separate function? Combine and make the template name a parameter.